targets/split_8mb4mb.mk

```diff
@@ -7,12 +7,24 @@
 #
 # When flashing via an external programmer it is easiest to have
 # two separate files for these pieces.
-all: $(board_build)/heads-$(BOARD)-$(HEADS_GIT_VERSION)-bottom.rom
-$(board_build)/heads-$(BOARD)-$(HEADS_GIT_VERSION)-bottom.rom: $(board_build)/$(CB_OUTPUT_FILE)
-	$(call do,DD 8MB,$@,dd of=$@ if=$< bs=65536 count=128 skip=0 status=none)
-	@sha256sum $@ | tee -a "$(HASHES)"
+all: bottom top
 
-all: $(board_build)/heads-$(BOARD)-$(HEADS_GIT_VERSION)-top.rom
-$(board_build)/heads-$(BOARD)-$(HEADS_GIT_VERSION)-top.rom: $(board_build)/$(CB_OUTPUT_FILE)
-	$(call do,DD 4MB,$@,dd of=$@ if=$< bs=65536 count=64 skip=128 status=none)
-	@sha256sum $@ | tee -a "$(HASHES)"
+bottom: $(board_build)/heads-$(BOARD)-$(HEADS_GIT_VERSION)-bottom.rom
```

Comment:
If you want, you could make a variable that holds `$(board_build)/heads-$(BOARD)-$(HEADS_GIT_VERSION)` because that's so long and repeated so many times, e.g. `BUILD_ROM_PREFIX=$(board_build)/heads-$(BOARD)-$(HEADS_GIT_VERSION)`. But not required at all, I'd be happy merging as-is.